app/deepseg.cc

```diff
@@ -568,6 +569,15 @@ int main(int argc, char* argv[]) try {
 	if (expWidth != vidGeo.value().first) {
 		fprintf(stderr, "Warning: virtual camera aspect ratio does not match capture device.\n");
 	}
+	// calculate crop region, only if result always smaller
+	if (expWidth != vidGeo.value().first &&
+		vidGeo.value().first <= capGeo.value().first &&
+		vidGeo.value().second <= capGeo.value().second) {
+			crop_region = calcCropping(capGeo.value().first,
+			                           capGeo.value().second,
+			                           vidGeo.value().first,
+			                           vidGeo.value().second);
```

Comment:
Avoid too long overhangs. Indentation should only increase one level, not those alignments to under the opening parens (which just look ugly).

```suggestion
			crop_region = calcCropping(
				capGeo.value().first, capGeo.value().second,
				vidGeo.value().first, vidGeo.value().second);
```

Having this repeated `var.value().field` kinda looks ugly, even though it is right semantically …